gossip/src/restart_crds_values.rs

```diff
@@ -122,6 +133,45 @@ impl RestartLastVotedForkSlots {
     }
 }
 
+impl Sanitize for RestartHeaviestFork {
+    fn sanitize(&self) -> Result<(), SanitizeError> {
+        sanitize_wallclock(self.wallclock)?;
+        self.last_slot_hash.sanitize()
+    }
+}
+
+impl RestartHeaviestFork {
+    pub fn new(
+        from: Pubkey,
+        now: u64,
+        last_slot: Slot,
+        last_slot_hash: Hash,
+        observed_stake: u64,
+        shred_version: u16,
+    ) -> Self {
+        Self {
+            from,
+            wallclock: now,
+            last_slot,
+            last_slot_hash,
+            observed_stake,
+            shred_version,
+        }
+    }
+
+    pub fn new_rand<R: Rng>(rng: &mut R, pubkey: Option<Pubkey>) -> Self {
```

Comment:
Does this need to be `pub`?
I feel like every review I have to ask not to pollute public interface.

Can you go over all `pub`s in this code and remove those not needed?
or at least change them to `pub(crate)`.

Reply:
This is used in tests, and I don't see any benchmarks needing it now. So I changed it to pub(crate).